examples/parametric/plot_equivalent_width.py

```diff
@@ -164,7 +172,7 @@ def equivalent_width(grids, uv_index, index_window, blue_window, red_window):
 
 
 if __name__ == "__main__":
-    grid_dir = "../../tests/test_grid"  # Change this directory to your own.
+    grid_dir = "../tests"  # Change this directory to your own.

```

Comment:
this should be relevant to the directory the script is in, or it will fail the PR tests

Reply:
Will change the directory path and be sure that all PR tests pass before creating another PR